content/en/docs/instrumentation/java/manual.md

```diff
@@ -486,6 +481,7 @@ public class DiceApplication {
         .setMeterProvider(sdkMeterProvider)
         .setLoggerProvider(sdkLoggerProvider)
         .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+        .setPropagators(ContextPropagators.create(W3CBaggagePropagator.getInstance()))
```

Comment:
Unfortunately this will simply override the previous setting; you have to do it the following way:
```suggestion
        .setPropagators(ContextPropagators.create(TextMapPropagator.composite(W3CTraceContextPropagator.getInstance(), W3CBaggagePropagator.getInstance())))
```